cmd/deployment/elasticsearch/keystore/set.go

```diff
@@ -0,0 +1,88 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cmdelasticsearchkeystore
+
+import (
+	"github.com/elastic/cloud-sdk-go/pkg/models"
+	"github.com/spf13/cobra"
+
+	cmdutil "github.com/elastic/ecctl/cmd/util"
+	"github.com/elastic/ecctl/pkg/deployment/elasticsearch"
+	"github.com/elastic/ecctl/pkg/ecctl"
+)
+
+const setKeystoreLong = `Manages the keystore settings of an Elasticsearch cluster.
+Note that the underlying API call uses the PATCH method, meaning that each operation is add/modify only.`
```

Comment:
not sure if we want to provide that level of details to the public. I mean we should clearly mention that this is `add/modify` and not replace but `API` and `PATCH` terms don't seem good as part of users help, no?